src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec

```diff
@@ -88,9 +89,12 @@
 
     <!-- net9.0 -->
     <file src="net9.0\build\common\MSTest.TestAdapter.props" target="build\net9.0\MSTest.TestAdapter.props" />
-    <file src="net9.0\build\common\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
+    <file src="net9.0\build\uwp\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
+    <file src="net9.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\net9.0\" />
     <file src="net9.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll" target="build\net9.0\" />
+    <file src="net9.0-windows10.0.17763.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll" target="build\net9.0\uwp" />
```

Comment:
How are we differentiating net9.0 UWP vs net9.0 WinUI? They both use the same TFM right?

Reply:
I believe the plan was to use `UseWinUI` and `UseUwpTools` properties.